blank-canvas-blocks/sass/blocks/_search.scss

```diff
@@ -0,0 +1,32 @@
+.wp-block-search {
+	&.wp-block-search__button-inside {
+		.wp-block-search__inside-wrapper{
+			border: var(--wp--custom--form--border--width) var(--wp--custom--form--border--style) var(--wp--custom--form--border--color);
+			border-radius: var(--wp--custom--form--border-radius);
+			.wp-block-search__input {
+				padding: var(--wp--custom--form--padding);
+			}
+			.wp-block-search__button {
+				padding: var(--wp--custom--search--button--padding);
+			}
+		}
+	}
+
+	.wp-block-search__input {
+		border: var(--wp--custom--form--border--width) var(--wp--custom--form--border--style) var(--wp--custom--form--border--color);
+		border-radius: var(--wp--custom--form--border--radius);
+		color: var(--wp--custom--form--color--text);
+		padding: var(--wp--custom--form--padding);
+		background: var(--wp--custom--form--color--background);	
+		font-size: var(--wp--custom--form--font-size);
+		line-height: var(--wp--custom--typography--line-height);
+	}
+
+	.wp-block-search__button {
+		@include button-main-styles;
```

Comment:
What do you think about including the file that the mixin is in, to make the dependencies explicit?